### ISSUE TITLE
[R1 Beta] Built-in support is too close (in X/Y) to the TFT35 bezel

### ISSUE THREAD

**Issue by AlejandroRivera** [PrintersForAnts/Micron#137]
While printing the [TFT35 bezel](https://github.com/PrintersForAnts/Micron/blob/8ec208c5365e73786e8c0e066155aa44c8fbe698/R1_Beta/STLs/Skirts/Displays/TFT35/%5Ba%5D_bezel_x1.stl), I noticed that the clearance with the built-in support wasn't enough to prevent the pieces from being fused together (at least in a Prusa MK4S printing in with PLA using Voron settings). 

![image](https://github.com/user-attachments/assets/239ceec6-4701-4aca-af7d-ec6217b1e258)

![image](https://github.com/user-attachments/assets/173e731a-7559-41ff-871d-3ba09290ae1a)

The support was so fused it had to be cut with a exacto knife. Here's a picture of the result to give you an idea of where the fused edges happened:
![image](https://github.com/user-attachments/assets/7cc8a6c1-c637-4e9c-b02f-21fba0bb56df)

I'd recommend increasing the clearance from 0.2mm to 0.3mm

**Comment by hartk1213** [comment]
ABS is the recommended filament to print these in, i havent tried it with PLA but with ABS it was no issue

**Comment by AlejandroRivera** [comment]
Yup, i agree that ABS is recommended and perhaps it wouldn't have this issue, or if it did, it might not be as hard to break apart, however, like you said, looking into it for a future bug fix might still help others since the Voron materials guide does say (https://docs.vorondesign.com/materials.html#pla) 

> [...] PLA is entirely acceptable for skirts and panel clips as they do not see the same high temperatures or forces.

**Comment by hartk1213** [comment]
@AlejandroRivera looking in cad the gap is actually 0.4mm 

![Image](https://github.com/user-attachments/assets/48633f88-7159-4500-ad2a-0e366d9e8ddd)

**Comment by hartk1213** [comment]
ah i see now, one is .4 the other is .2 ill get that fixed 

![Image](https://github.com/user-attachments/assets/4dd1f30d-eee1-4e10-acbd-763b96aa79be)

**Comment by AlejandroRivera** [comment]
Thanks!
